test/unicode-spec.js

```diff
@@ -87,8 +97,8 @@ describe("unicode", function() {
         expect('\\text{여보세요}').toParse();
     });
 
-    it("should not parse CJK outside \\text{}", function() {
-        expect('私はバナナです。').toNotParse();
-        expect('여보세요').toNotParse();
+    it("should parse CJK outside \\text{}", function() {
+        expect('私はバナナです。').toParse();
+        expect('여보세요').toParse();
```

Comment:
This change isn't really related to accents and should probably be in a separate PR.

Reply:
It's relevant in that this PR rewrites how we parse Unicode. The relevant line change is from `if (this.mode === "text" && cjkRegex.test(nucleus.text))` to `if (cjkRegex.test(text))`. I can't make a separate PR now because they'd both change that line, but if you like, I can remove that change from here and add a second PR once this one is accepted.

Reply:
> I can remove that change from here and add a second PR once this one is accepted.

I think that makes the most sense.  Thanks. :)